### PR TITLE
Fixes statue simplemob teleport not working and 3 other spells not appearing

### LINKED ISSUE #65583: The "Teleport" ability of the alive statue mob doesn't work.
It finishes the do after process and then fails to do its' job.

### PR DESCRIPTION
**Pull request by SmArtKar**
## About The Pull Request

Closes #65583
Also fixed 3 other spells(nightvision, aoe blind and night vision) not appearing

## Changelog

:cl:
fix: Fixed statue simplemob teleport not working and 3 other spells not appearing
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/hostile/statue.dm [hunk 1]
```diff
@@ -94,68 +97,39 @@
...
-/mob/living/simple_animal/hostile/statue/face_atom()
+/mob/living/simple_animal/hostile/netherworld/statue/face_atom()
 	if(!can_be_seen(get_turf(loc)))
 		..()
 
-/mob/living/simple_animal/hostile/statue/IsVocal() //we're a statue, of course we can't talk.
+/mob/living/simple_animal/hostile/netherworld/statue/IsVocal() //we're a statue, of course we can't talk.
 	return FALSE
 
-/mob/living/simple_animal/hostile/statue/proc/can_be_seen(turf/destination)
```
Comment (SuperNovaa41) [inline code/modules/mob/living/simple_animal/hostile/statue.dm:116]: why did you remove this?

**Review by Maurukas — APPROVED** [review]
Maps are good. Just a repath.
